sdk/src/main/java/com/uid2/UID2Manager.kt

```diff
@@ -93,7 +93,7 @@ public class UID2Manager internal constructor(
     private val inputUtils: InputUtils,
     defaultDispatcher: CoroutineDispatcher,
     initialAutomaticRefreshEnabled: Boolean,
-    private val logger: Logger,
+    @property:InternalUID2Api public val logger: Logger,
```

Comment:
This is what we did in OpenPass, to allow other modules to access the configured `Logger` but to hide it from normal public consumption.